src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -48,42 +51,51 @@ public Command parseCommand(String userInput) throws ParseException {
         final String arguments = matcher.group("arguments");
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
+        case AddCommand.COMMAND_WORD: case AddCommand.COMMAND_ALIAS:
```

Comment:
Should have a new line for each new case instead of putting both of these in the same line.